components/inputs/demo/input-number.html

```diff
@@ -98,6 +98,18 @@ <h2>Trailing Zeroes</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Denominator</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-input-number max="5" min="0" label="Grade" label-hidden required unit="/5" unit-label="out of 5" value="3"></d2l-input-number>
+					<style>
+						d2l-input-number {
+							--d2l-input-text-align: end;
+						}
```

Comment:
@tdevos-d2l this is how you can right-align the input value text.

Reply:
I think you made *all* the input number demos right-aligned with this 😄

Reply:
🤦 yes, yes I did.